templates/zerver/help/send-a-private-message.md

```diff
@@ -1,32 +1,32 @@
-# Private Message
+# Send a private message
```

Comment:
I'm a tad confused - we also have a `Send someone a private message` in `People` which has no docs - should we reference this doc there too? Put this doc in the `People` section?